x-pack/plugins/synthetics/public/legacy_uptime/components/fleet_package/synthetics_policy_edit_extension_wrapper.tsx

```diff
@@ -110,32 +29,17 @@ export const SyntheticsPolicyEditExtensionWrapper = memo<PackagePolicyEditExtens
           </EuiButton>
         </EuiCallOut>
       );
+    } else {
+      return (
+        <EuiCallOut>
+          <p>{EDIT_DISABLE_DESC}</p>
+          {/* TODO Add a link to exact monitor*/}
+          <EuiButton href={`${http?.basePath.get()}/app/uptime/manage-monitors/all`}>
```

Comment:
I think we should be directing people to Synthetics, not Uptime, if I'm not mistaken.